content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -66,6 +66,13 @@ of the form `auto-generated-name.my-svc.my-namespace.svc.cluster-domain.example`
 
 ## Pods
 
+### A/AAAA records
+
+If pods created by associated `Deployment` or `DaemonSet` then the following
+schema of DNS resolution will be in place
```

Comment:
```suggestion
Any pods created by a `Deployment` or `DaemonSet` will have the following
DNS resolution available:
```

Reply:
Actually can I suggest a further refinement:
```suggestion
Any pods created by a Deployment or DaemonSet have the following
DNS resolution available:
```

The [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) recommends not putting resource names in backticks.